data/json/items/tool/workshop.json

```diff
@@ -881,6 +883,23 @@
       }
     ]
   },
+  {
+    "id": "tire_iron",
+    "type": "TOOL",
+    "name": { "str": "tire iron" },
+    "description": "A steel rod bent 90 degrees with a socket on the shorter end, made to hold the lug nuts of a car's wheels to either fasten or loosen for convenient removal and replacement.  Makes for a decent melee weapon in a pinch.",
+    "weight": "454 g",
+    "longest_side": "46 cm",
+    "volume": "1500 ml",
+    "price": 30,
+    "price_postapoc": 50,
```

Comment:
Maybe increase price a bit? `50` means it costs mere 50 cents, that's too little IMO for such a handy tool, esp. for post-apocalypse.

Reply:
I bumped up the price from 50 cents to around 2 bucks. It sorta shocks me that the wrench is only a single dollar in post-apoc price, though

Reply:
I think the price should be like ten times higher. Low cost of wrench is an oversight/error, and it should be fixed too.

Reply:
Right, that's fair

Reply:
Made it 20 bucks post apoc now